app/move/views/view.njk

```diff
@@ -204,7 +204,7 @@
     }) }}
   {% endif %}
 
-  {% if canAccess("move:cancel") and move.status == "requested" and not move.allocation %}
+  {% if (canAccess("move:cancel") and move.status == "requested" and not move.allocation) or (canAccess("move:cancel:proposed") and move.status == "proposed") %}
```

Comment:
I have no idea what this is going to do because I have no idea how the nunjucks operators figure out precedence.

Is it possible to do this check in the controller and set an explicit local var to use?

Reply:
I am not terribly sure it's desirable as it's completely presentational. Maybe I can try and make it more readable.